tools/lib/capitalization.py

```diff
@@ -109,6 +109,9 @@
     # Use in compose box.
     r"to send",
     r"to add a new line",
+    # Used in user deactivation modal
+    r"does not administer any active bots.",
+    r"administers the following active bots:",
```

Comment:
I think you can drop this hunk, since we're no longer using the strings involved.